example/android/app/build.gradle

```diff
@@ -124,7 +127,6 @@ android {
 dependencies {
     // The version of react-native is set by the React Native Gradle Plugin
     implementation("com.facebook.react:react-android")
-    implementation("com.facebook.react:flipper-integration")
```

Comment:
Guess there was some compatibility issue?

Reply:
I believe flipper support was fully removed from react native in 74 or 75. We weren't really using it anyway.